_sass/governance.scss

```diff
@@ -0,0 +1,17 @@
+.trustee {
+  &-row {
```

Comment:
While using `&` like this to join within a word is clever, I'd discourage doing so as it's clearer not to. The duplication of `trustee` in `trustee-row` and `trustee-avatar` is pretty minimal and allows for better discoverability (by searching for the exact class name).

Reply:
👍 29ce178